### DISCUSSION TITLE
WSL2: Caddy https not working

### DISCUSSION THREAD

**Discussion by Basti271** [dunglas/symfony-docker#363] [Q&A]
Hi, 
I use WSL 2 and followed the instructions from the docs running this command in Linux:

`$ docker cp $(docker compose ps -q caddy):/data/caddy/pki/authorities/local/root.crt /usr/local/share/ca-certificates/root.crt && sudo update-ca-certificates`

The root.crt file exists as well as the root.crt, but I still lack https when accessing my localhost. Are there any further steps I should make?

**Comment by maxhelias** [comment]
I think you need to extract the certificate from your WSL to allow it on your Windows machine
